components/org.wso2.carbon.secvault.securevault/src/main/java/org/wso2/carbon/secvault/securevault/MasterKey.java

```diff
@@ -14,15 +14,15 @@
  * limitations under the License.
  */
 
-package org.wso2.carbon.securevault;
+package org.wso2.carbon.secvault.securevault;
 
 import java.util.Optional;
 
 /**
  * This class represents a master key that is needed to initialize a secret repository. A secret repository might need
  *  a few master keys to get it initialized.
  *
- * @since 5.2.0
+ * @since 1.0.0
```

Comment:
Yes, that might be an option as this is a new repository, however I think that might be wrong as this component has been released earlier while in the carbon-kernel repository:
https://mvnrepository.com/artifact/org.wso2.carbon/org.wso2.carbon.securevault